beanstalk_worker/worker_start_test.py

```diff
@@ -208,7 +208,17 @@ def test_job_data(job_data):
     try:
         job = bs.reserve()
         job_data = json.loads(job.body)
-        test_job_data(job_data)
+        if len(sys.argv) > 1 and sys.argv[1] == "ci":
+            # If working in CI environment, don't run the scanner
+            logger.log(level=logging.INFO, msg="CI check...")
+            job_data["action"] = "start_delivery"
+            job_data["msg"] = ""
+            job_data["logs"] = ""
```

Comment:
>logs are otherwise sent as JSON, if logs are empty, we should return {} instead of empty string.

You suggest we use `{}` instead of `""`? Ack, works for me, I can change it.

> Although both values can fail the if condition, we can be consistent in the type of value field for every key.

But if value for one key is a list and for other it's a string, how do we keep consistency in the overall json? Maybe I'm missing something here. How about we use `None` for empty values?